library/src/main/java/com/otaliastudios/zoom/ZoomEngine.kt

```diff
@@ -1336,23 +1351,22 @@ internal constructor(context: Context) : ViewTreeObserver.OnGlobalLayoutListener
      * @param zoomTargetX     the x-axis zoom target
      * @param zoomTargetY     the y-axis zoom target
      */
+    @SuppressLint("ObjectAnimatorBinding")
     private fun animateZoomAndAbsolutePan(@Zoom zoom: Float,
                                           @AbsolutePan x: Float, @AbsolutePan y: Float,
                                           allowOverScroll: Boolean,
                                           allowOverPinch: Boolean = false,
                                           zoomTargetX: Float? = null,
                                           zoomTargetY: Float? = null) {
         if (setState(ANIMATING)) {
-            mClearAnimation = false
             @Zoom val startZoom = this.zoom
             @Zoom val endZoom = checkZoomBounds(zoom, allowOverScroll)
             val startPan = pan
             val targetPan = AbsolutePoint(x, y)
             LOG.i("animateZoomAndAbsolutePan:", "starting.", "startX:", startPan.x, "endX:", x, "startY:", startPan.y, "endY:", y)
             LOG.i("animateZoomAndAbsolutePan:", "starting.", "startZoom:", startZoom, "endZoom:", endZoom)
 
-            @SuppressLint("ObjectAnimatorBinding")
-            val animator = ObjectAnimator.ofPropertyValuesHolder(mContainer,
+            ObjectAnimator.ofPropertyValuesHolder(mContainer,
```

Comment:
Can you remove mContainer from this constructor?

As far as I remember, It might be useless and harmful. I think ObjectAnimator will try to call `mContainer.setPan()` and `mContainer.setZoom()` with reflection (failing to do so).

Reply:
Ohh so I can just remove the parameter completely! Yes that makes sense.
I used this kind of "multiple value animations" animator the first time there. Didn't think it though back then.